src/Routers/SchemasRouter.js

```diff
@@ -14,16 +14,32 @@ function classNameMismatchResponse(bodyClass, pathClass) {
   );
 }
 
+function injectDefaultSchema(schema) {
+  let defaultSchema = Schema.defaultColumns[schema.className];
+  if (defaultSchema) {
+    Object.keys(defaultSchema).forEach((key) => {
+      schema.fields[key] = defaultSchema[key];
+    });
+  }
+  return schema;
+}
+
 function getAllSchemas(req) {
   return req.config.database.schemaCollection()
     .then(collection => collection.getAllSchemas())
+    .then(schemas => {
```

Comment:
If you want to be more concise, you could do

`then(schemas => schemas.map(injectDefaultSchema))`

Up to you though.